rpc/resource/resource.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package infoblox.rpc;
```

Comment:
Should this be infoblox.rpc or atlas.rpc?

Reply:
Hmm..., I don't know